Sources/_StringProcessing/RegexDSL/DSL.swift

```diff
@@ -62,6 +62,44 @@ extension CharacterClass: RegexProtocol {
 
 // Note: Quantifiers are currently gyb'd.
 
+/// Specifies how much to attempt to match when using a quantifier.
+public struct QuantificationKind {
```

Comment:
- "Kind" feels a bit too general. Kind could also refer to one-or-more/zero-or-more/zero-or-one. Is there a more specific word for this? Maybe `QuantificationBehavior`?
- For the DSL, should quantifier kinds read like an adverb for fluency?
  ```swift
  oneOrMore("x", .eagerly)
  oneOrMore("x", .reluctantly)
  ```

Reply:
Agreed on both these counts — good call!

Reply:
Would these instead be properties returning `Self` if we're going that route?